src/components/interaction-designer/block-editors/MaxDigitEditor.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="hasIvr" class="form-group block-max-digits">
+  <div v-if="hasIvr" class="form-group block-max-digits" :id="`${block.uuid}.config.ivr.maxDigits`">
```

Comment:
Let's be aware of these casing, if vmo-3553 (consume flow-runner v1.0) gets merged before this PR, we need to change the case here and in similar location.

Reply:
Thanks for pointing this out. I will be consuming the IDs as part of [VMO-3478](https://viamoinc.atlassian.net/browse/VMO-3478). I will keep an eye out for these changes 👍